### PR TITLE
feat: add home screen html and css

### PR DESCRIPTION
**Pull request by abhishek-bhatkar**

### CODE REVIEW

File: css/base.css [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+
+  .fs-xs {
+    font-size: 0.5rem;
+  }
+  
+  .fst-italic {
+    font-style: italic;
+  }
+
+  :root{
```
Comment (khushijohri2001) [inline css/base.css:30]: You can keep the root at the beginning of the code

File: css/base.css [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  .fs-xs {
+    font-size: 0.5rem;
+  }
+  
+  .fst-italic {
+    font-style: italic;
+  }
+
+  :root{  
+  --box-shadow-card: 0px 3px 18px -5px #d1d5db;
```
Comment (khushijohri2001) [inline css/base.css:31]: You can use a variable name for the color here. It will help you to keep your code consistent.

**Review by khushijohri2001 — APPROVED** [review]
Otherwise the code looks good to go

File: css/home.css [hunk 3]
```diff
@@ -0,0 +1,168 @@
...
+.flex-gap-lg {
+    gap: 2rem;
+}
+.flex-items-center {
+    align-items: center;
+}
+.flex-justify-center {
+    justify-content: center;
+}
+.flex {
```
Comment (ckahub) [inline css/home.css:102]: avoid giving class name same as property(flex)

File: css/home.css [hunk 4]
```diff
@@ -0,0 +1,168 @@
...
+  
+  .banner-img .img {
+    height: 70vh;
+  }
+  
+  .banner-info .btn {
+    padding: 0.75rem 4rem;
+  }
+  
+  .banner-info .btn:hover {
```
Comment (ckahub) [inline css/home.css:23]: why created hover ,when you are not defining boxshadow

**Review by JaswanthReddyPadala — COMMENTED** [review]
Good Class Names, Code looks Clean ✨
Good to Merge ✔

**Pull request merged**